src/status_im2/contexts/onboarding/events.cljs

```diff
@@ -111,12 +111,14 @@
 
                          :currentNetwork              config/default-network
                          :previewPrivacy              config/blank-preview?}]
-    {effect    request
-     :dispatch [:navigate-to :generating-keys]
-     :db       (-> db
-                   (dissoc :multiaccounts/login)
-                   (dissoc :auth-method)
-                   (assoc :onboarding-2/new-account? true))}))
+    {effect          request
+     :dispatch       [:navigate-to :generating-keys]
```

Comment:
@briansztamfater is this `:dispatch` needed? 🤔

Reply:
I think it is, we immediately navigate to `:generating-keys` screen and then dispatch `:onboarding-2/navigate-to-identifiers` with delay.